jerry-libm/log.c

```diff
@@ -105,11 +107,11 @@ log (double x)
   {
     if (((hx & 0x7fffffff) | lx) == 0) /* log(+-0) = -inf */
     {
-      return -two54 / zero;
+      return -INFINITY;
     }
     if (hx < 0) /* log(-#) = NaN */
     {
-      return (x - x) / zero;
+      return NAN;
```

Comment:
If jerry-libm gets disabled for MSVC by default, this change will become unnecessary.

Reply:
Yeap , I wanna libm could be enable in msvc, then we can create very tiny jerryscript engine (without msvc runtime depends, that's a dependence hell, if you have experience with it)

Reply:
Fixed. LIBC & LIBM used under Win32.